routes/teams.js

```diff
@@ -105,8 +104,8 @@ var createQuiz = function(rb_team_id, list_id, league, quiz_name, res, callback,
 router.get('/team', function(req, res) {
     var term = req.query.q;
     //find the team, This syntax does a sql-type like clause with case insensitivity with the RegEx
-    db.collection('teams').find({$or: [{'name': new RegExp(term, 'i')}, {'market': new RegExp(term, 'i')}, {'keywords': new RegExp(term, 'i')}]}).sort({'market': 1}).toArray(function (err, items) {
-    	res.json(items);
+    db.collection('teams').find({$or: [{'name': new RegExp(term, 'i')}, {'market': new RegExp(term, 'i')}, {'category': new RegExp(term, 'i')}, {'keywords': new RegExp(term, 'i')}]}).sort({'market': 1}).toArray(function (err, items) {
```

Comment:
just a thought, but i dont like having to scan both category + keywords. I think we should just stop using the keywords field since it is redundant, right?